src/jobs/integration-test.yaml

```
@@ -0,0 +1,43 @@
+description: |
+  Runs an integration test by creating a KIND cluster and executing it as a Go
+  test.
+
+  Uses the machine executor and replaces the Go version with a version
+  controlled by us since the preinstalled Go is very outdated.
+
+  Tests can be customized by providing a KIND config file or a setup script
+  which is run before the test is executed.
+environment:
+  GOPATH: /go
+  KUBERNETES_VERSION: v1.16.3
+executor: machine
+parameters:
+  env-file:
+    default: ".env"
+    description: "File of environment variables to set."
+    type: string
+  kind-config:
+    default: ".kind-config"
+    description: "Path to kind config file."
+    type: string
+  setup-script:
+    default: "setup.sh"
```

This should be empty by default so we don't trigger setup commands with non-existing path by default.

```suggestion
    default: ""
```

---

Yes good shout. Fixed.